src/formats/wkt/WktParser.js

```diff
@@ -0,0 +1,120 @@
+     * var parser = new WktParser('POINT (19 23)');<br/>
+     * parser.load(null, function(shape) {<br/>
+     *   if(shape.type == WktType.SupportedGeometries.POINT) {<br/>
+     *     var shapeAttributes = new ShapeAttributes(null);<br/>
+     *     shapeAttributes.fontColor = Color.RED;<br/>
+     *     return {<br/>
+     *         attributes: shapeAttributes<br/>
+     *     };<br/>
+     *   }<br/>
+     * }, layer);<br/>
+     * wwd.addLayer(layer);<br/>
+     *
+     * @param textRepresentation {String} Text representation of WKT objects.
+     * @constructor
+     * @alias WktParser
+     */
+    var WktParser = function (textRepresentation) {
+        this.objects = null;
+
+        this.textRepresentation = textRepresentation;
+    };
+
+    /**
+     * It parses the received string and create the Objects, which then can be rendered.
+     * @param parserCompletionCallback {Function} An optional function called when the WKT loading is
+     *   complete and all the shapes have been added to the layer.
+     * @param shapeConfigurationCallback {Function} This function  is called whenever new shape is created. It provides
+     *   the current shape as the first argument. In this way it is possible to modify the shape even provide another one.
+     *   If any shape is returned it is used in place of the previous one. This function should be synchronous and if
+     *   you want to provide custom shape, it has to be synchronous.
+     * @param layer {RenderableLayer} Layer to use for adding all the parsed shapes
+     * @return {Renderable[]} Array of the Renderables present in the WKT.
+     */
+    WktParser.prototype.load = function (parserCompletionCallback, shapeConfigurationCallback, layer) {
+        var objects = new WktTokens(this.textRepresentation).objects();
+
+        shapeConfigurationCallback = shapeConfigurationCallback || this.defaultShapeConfigurationCallback;
+        parserCompletionCallback = parserCompletionCallback || this.defaultParserCompletionCallback;
+
+        var shapes = [];
+        objects.forEach(function(object){
+            object.shapes().forEach(function(shape){
+                var configuration = shapeConfigurationCallback(object);
+                if(configuration && configuration.attributes) {
+                    shape.attributes = configuration.attributes;
+                }
+                if(configuration && configuration.highlightAttributes) {
+                    shape.highlightAttributes = configuration.highlightAttributes;
+                }
+                if(configuration && configuration.pickDelegate) {
+                    shape.pickDelegate = configuration.pickDelegate;
+                }
+                if(configuration && configuration.userProperties) {
+                    shape.userProperties = configuration.userProperties;
+                }
+                shapes.push(shape);
+            });
+        });
+
+        layer.addRenderables(shapes);
```

Comment:
Shapefile performs the step of adding shapes to the layer in its defaultParserCompletionCallback. It appears that GeoJSONParser does not. We need to define a consistent pattern. @balhar-jakub and @AkeluX what are your opinions about this? Can you think of a good reason why Shapefile might implement this step in the default callback?

Reply:
The reason I can see is that it makes it relatively simple to ignore the viewing logic and use it just as a parser instead of also showing it.

If we want to pursue this direction and allow usage as just parsers, it would make more sense to me to stop passing them layers and instead return a promise which will return all the shapes either as the Renderables or as objects relevant to the parser. I would go with the Renderables. 

What is you opinion @AkeluX ?

Reply:
Although I like and agree with the proposition from Jakub for the long run, I think considering promises and not passing a layer is out of the scope of this pull request.

The approach implemented for Shapefile actually allows using the class as a pure parser without creating renderables by overriding the default completion callback. I would follow this approach for WKT and create an issue to adapt the GeoJSONParser as well.

Reply:
This sounds good to me. Let's proceed with the plan @AkeluX has outlined here.

Our strategy for adopting modern JavaScript conventions in our interfaces - such as promises - is a good topic of discussion for our team meeting this month.